### ISSUE TITLE
[need information] help to find the detail format description for pinballs (those files generated from Pinplay/SDE)?

### ISSUE THREAD

**Issue by melodylail** [intel/pinball2elf#13]
I am trying to understand the detail workflow in pinball2elf by reading code and test it on speccpu2017.
Asi encount some unkonw issues, i try to debug it and find that miss detail format information about those Pinball file( xxx.sel, xxx.address, etc).
Could you please guide me where can i found those format information?

**Comment by hgpatil** [comment]
[Pinballs: Portable and Shareable User-level Checkpoints for Reproducible Analysis and Simulation](https://wiki.ith.intel.com/download/attachments/11733762/pinballs-abstract.pdf?version=1&modificationDate=1391602571747&api=v2); Harish Patil (Intel Corporation), Trevor E. Carlson (Ghent University).

**Comment by melodylail** [comment]
@hgpatil thanks for your information. But i still can not find those detail format information for those pinball files.
After read this paper and try to find those "log format version" in the pinkit folder of SDE(recently released v9.x.x), unfortunately, no detail format information can be found. 
PS: 
i can find the "log format version: 2.16" in xxx.global.log(generated byh SDE 9.0.0). but can't figure out the detail format spec for this "log format version: 2.16".
Older version pinballs（for example，2.6）, cannot be handled correctly by Pinball2elf.

**Comment by hgpatil** [comment]
Yes, the pinball documentation I pointed to is at least 10 years old. It does describe various files accurately and the overall format has not changed over the years but some new files since have been added. Which files do you see un-documented and would like to know the format of? The choice to leave of really old pinball formats was deliberate.
